src/Resources/chunk-processor.js

```diff
@@ -0,0 +1,94 @@
+module.exports = function ()  {
```

Comment:
you care not very consistent with wrapping the JS in a closure or not...

Reply:
The other classes are defined as singletons. In either case no variable is leaked to global scope.